File: python/raindrops/raindrops.py
def convert(number):

    raintext = "" if number % 3 > 0 else "Pling"

    if number % 5 == 0: raintext += "Plang"

    if number % 7 == 0: raintext += "Plong"

    return str(number) if raintext == "" else raintext

File: r/raindrops/raindrops.R
raindrops <- function(number) {

    if (number %% 3) { txt = "" } else { txt = "Pling"}

    if (!number %% 5) { txt = paste(txt, "Plang", sep="") }

    if (!number %% 7) { txt = paste(txt, "Plong", sep="") }

    if (txt == "") { 
        return(as.character(number))
    } else {
        return(txt)
    }
}
